Shared/Utils/Data+MultisigExtension.swift

```diff
@@ -8,6 +8,39 @@ import CryptoSwift
 // MARK: - Hex String to Data conversion
 public extension Data {
 
+    static func value(of nibble: UInt8) -> UInt8? {
+        guard let letter = String(bytes: [nibble], encoding: .ascii) else { return nil }
+        return UInt8(letter, radix: 16)
+    }
+
+    // TODO: Duplicate code. Remove and use init(hex: String) instead, when possible
+    init(hexWC: String) {
```

Comment:
what's hexWC stands for?

Reply:
WalletConnect. This is the implementation that WalletConnect pulls in.